env.setup

```diff
@@ -0,0 +1,136 @@
+APP_ENV=production
```

Comment:
You might want to remove this file. It's not necessary to copy the .env template to the root directory, because the setup script handles those tasks.